cluster/examples/workloads/kubernetes/wordpress/aws/gitops/aws-infra/rds-class.yaml

```diff
@@ -3,7 +3,10 @@ apiVersion: database.aws.crossplane.io/v1alpha2
 kind: RDSInstanceClass
 metadata:
   name: standard-rds-mysql
+  labels:
+    example: "true"
 specTemplate:
+  writeConnectionSecretsToNamespace: crossplane-system
```

Comment:
@negz is this required?

Reply:
It is - it configured which namespace managed resources dynamically provisioned using this class write their connection secrets to. We considered implicitly using the namespace the provider credentials were stored in, but this seemed less magic, more flexible, and simpler to implement.